yoyodyne/models/expert.py

```diff
@@ -94,6 +95,12 @@ def encode_unseen_action(self, action: actions.Edit) -> int:
 
         Operates same as encode_action but does not expand lookup table.
         """
+        if action not in self.w2i:
+            util.log_info(
+                f"""Action {action} passed to SED expert is invalid.
+                Using random action instead. Check training set and index."""
```

Comment:
Can you shorten this into a single sentence? Maybe just "Action {action} is invalid; sampling random action". (None of our other log statements end in a period, so I have removed that.) The instruction to "check" is right now too vague, and explaining what to check and what to do about it is going to be too verbose for a log statement, so move it to a comment or the docs if you think it's important.

Reply:
Mmm, if this is too long for a docstring, I may just go back to it throwing an error. It really shouldn't be something that gets ignored during training.